mailbox/tools/indexer/src/test/java/org/apache/mailbox/tools/indexer/ReIndexerImplTest.java

```diff
@@ -98,7 +98,7 @@ void reIndexAllShouldCallMessageSearchIndex() throws Exception {
     }
 
     @Test
-    void reIndexMailboxPathShouldCallMessageSearchIndex() throws Exception {
+    void reIndexAllShouldCallMessageSearchIndex() throws Exception {
```

Comment:
Unrelated changes?

Reply:
Test renaming... I saw some tests with wrong names along the way and included a quick fix while I was at it. Is this a problem?